CMakeLists.txt

```
@@ -322,6 +322,8 @@ option(ENABLE_REMOTE_DISCOVERY_AVAHI "Enable remote disocvery in remote device f
 
 option(ENABLE_REMOTE_DISCOVERY_DHT "Enable remote disocvery in remote device for linux using DHT" OFF)
 
+option(ENABLE_REMOTE_DISCOVERY_ANDROID "Enable remote disocvery support in remote device for Android" OFF)
```

```suggestion
option(ENABLE_REMOTE_DISCOVERY_ANDROID "Enable remote discovery support in remote device for Android" OFF)
```

See also above option with the same typo.

---

I can see this option being useful also outside of android